src/actions/types.js

```diff
@@ -12,6 +12,7 @@ export type LoadUserOptionsActionType = {
 export type SaveGlobalOptionsActionType = {
   type: C.SAVE_GLOBAL_OPTIONS,
   payload: {|
+    autoconnect?: string,
```

Comment:
autoconnect => autoConnect

Reply:
I think in this context `autoconnect` as-is works ok.  No need to change.